aircraft/cost.py

```diff
@@ -0,0 +1,112 @@
+"""Aircraft cost models"""
+from gpkit.shortcuts import Var, Model
+import gpkit
+gpkit.disable_units()
```

Comment:
That's fine, then! Maybe create them as constants, if you think users might want to see how much their solutions rely on this fits?